taskt/Core/Automation/Commands/Excel Commands/ExcelWriteCellCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.Office.Interop.Excel;
+using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
+using Application = Microsoft.Office.Interop.Excel.Application;
+
+namespace taskt.Core.Automation.Commands
+{
+    [Serializable]
+    [Group("Excel Commands")]
+    [Description("This command sets the value of a specific cell in an Excel Worksheet.")]
+
+    public class ExcelWriteCellCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [PropertyDescription("Excel Instance Name")]
+        [InputSpecification("Enter the unique instance that was specified in the **Create Application** command.")]
+        [SampleUsage("MyExcelInstance || {vExcelInstance}")]
+        [Remarks("Failure to enter the correct instance or failure to first call **Create Application** command will cause an error.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_InstanceName { get; set; }
+
+        [XmlAttribute]
+        [PropertyDescription("Cell Value")]
+        [InputSpecification("Enter the text value that will be set in the selected cell.")]
+        [SampleUsage("Hello World || {vText}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_TextToSet { get; set; }
+
+        [XmlAttribute]
+        [PropertyDescription("Cell Location")]
+        [InputSpecification("Enter the location of the cell to extract.")]
```

Comment:
Change to "Enter the location of the cell to set the text value."

Reply:
done